src/plugins/vis_type_table/public/table_vis_fn.ts

```diff
@@ -35,26 +10,29 @@ import {
   OpenSearchDashboardsDatatable,
   Render,
 } from '../../expressions/public';
-import { VisRenderValue } from '../../visualizations/public';
+import { TableVisConfig } from './types';
 
 export type Input = OpenSearchDashboardsDatatable;
 
 interface Arguments {
   visConfig: string | null;
 }
 
-interface RenderValue extends VisRenderValue {
+export interface TableVisRenderValue {
   visData: TableContext;
   visType: 'table';
+  visConfig: TableVisConfig;
 }
 
-export const createTableVisFn = (): ExpressionFunctionDefinition<
-  'opensearch_dashboards_table',
+export type TableVisExpressionFunctionDefinition = ExpressionFunctionDefinition<
+  'visTable',
```

Comment:
I dont think we should rename this function. If someone is relying on the table expression, this change will be a breaking change for them.

Reply:
sure. I will name it back.